launch.go

```diff
@@ -187,72 +187,99 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 	}
 
 	log.Printf("Fetching Build %d", buildID)
-	b, err := api.BuildFromID(buildID)
+	build, err := api.BuildFromID(buildID)
 	if err != nil {
 		return fmt.Errorf("Fetching Build ID %d: %v", buildID, err)
 	}
 
-	log.Printf("Fetching Job %d", b.JobID)
-	j, err := api.JobFromID(b.JobID)
+	log.Printf("Fetching Event %d", build.EventID)
+	event, err := api.EventFromID(build.EventID)
```

Comment:
We only need to make this call if there is no parent build

Reply:
Fixed 👍